### PR TITLE
feat(replica): snapshot space information for replicas

### PR DESCRIPTION
**Pull request by mtzaurus**
V1 ReplicaSpaceUsage now also contains information about space used by all existing replica snapshots.

### CODE REVIEW

**Comment by mtzaurus** [comment]
bors try

File: io-engine/src/grpc/v1/replica.rs [hunk 1]
```diff
@@ -76,11 +76,12 @@ impl From<LvolSpaceUsage> for ReplicaSpaceUsage {
             allocated_bytes: u.allocated_bytes,
             // todo: rpc api submodule was updated, so have to fill these in as
             // 0 for now..
-            allocated_bytes_snapshots: 0,
             cluster_size: u.cluster_size,
             num_clusters: u.num_clusters,
             num_allocated_clusters: u.num_allocated_clusters,
-            num_allocated_clusters_snapshots: 0,
+            allocated_bytes_snapshots: u.allocated_bytes_snapshots,
```
Comment (tiagolobocastro) [inline io-engine/src/grpc/v1/replica.rs:80]: remove the todo comment above :D

**Review by tiagolobocastro — APPROVED** [review]

File: io-engine/src/grpc/v1/replica.rs (hunk 1 above)
Comment (mtzaurus) [reply io-engine/src/grpc/v1/replica.rs:80]: Indeed. :-)
Done.

**Comment by mtzaurus** [comment]
bors try

**Review by hrudaya21 — APPROVED** [review]

**Comment by mtzaurus** [comment]
bors merge

**Pull request merged**